test/cql-pytest/test_bloom_filter.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+import pytest
+import random
+import api_util
+import nodetool
+from util import new_test_table
+
+# Test inserts `N` rows into table, flushes it 
+# and tries to read `M` non-existing keys.
+# Then bloom filter's false-positive ratio is checked.
+@pytest.mark.parametrize("N,M,fp_chance", [(500, 1000, 0.1)])
+def test_bloom_filter(scylla_only, cql, test_keyspace, N, M, fp_chance):
+    keys = random.sample(range(2147483647), N+M)
+    with new_test_table(cql, test_keyspace, "a int PRIMARY KEY", 
+        f"WITH bloom_filter_fp_chance = {fp_chance}") as table:
+        
+        stmt = cql.prepare(f"INSERT INTO {table} (a) VALUES(?)")
+        for k in range(N):
+            cql.execute(stmt, [k])
+        nodetool.flush(cql, table)
+        
+        for k in range(N, N+M):
+            cql.execute(f"SELECT * FROM {table} WHERE a = {k} BYPASS CACHE")
+
+        fp = api_util.get_column_family_name_metric(cql, 
+          "bloom_filter_false_positives", table)
```

Comment:
Very nice that we have an API to read a per-table metric. I didn't even know we had such a feature. It solves the parallel test issue (which we don't do now, but may do in the future) nicely.